src/features/commands/psc.js

```diff
@@ -0,0 +1,248 @@
+  
+      const results = [];
+      for (let i = 0; i < items.length; i++) {
+        const item = items[i]
+        if (item.name.toLowerCase().includes(focusedValue.toLowerCase()) || item.id.toLowerCase().includes(focusedValue.toLowerCase())) {
+          results.push({
+            name: item.name,
+            value: item.id,
+          });
+        }
+      }
+  
+      // remove elements after 25
+      results.splice(25);
+  
+      await interaction.respond(results);
+    },  
+}
+
+
+async function handleStatusCommand(client, interaction) {
+  await interaction.reply("Loading...")
+
+  const embed = new EmbedBuilder()
+      .setColor(config.color)
+      .setTitle("Status")
+      .setFooter({ text: "/v1/status" })
+  const response = await (await requestPSC("/v1/status", interaction.member.user.tag)).json()
+
+  embed.addFields({ name: "Status:", value: `Success: \`\`${response.success}\`\``})
+
+  await interaction.editReply({ embeds: [embed] })
+}
+
+async function handleSkyblockPlayerCommand(client, interaction) {
+  await interaction.reply("Loading...")
+  // Gets the parameters object
+  const parameters = interaction.options
+  const username = parameters.get("username").value
+  await interaction.editReply(`Loading data for ${username}`)
+  const uuid = (await (await fetch(`https://mowojang.matdoes.dev/users/profiles/minecraft/${username}`)).json()).id
+  await interaction.editReply(`Loading data for ${username} (uuid: ${uuid})`)
+  const playerData = await (await requestPSC(`/v1/hypixel/skyblockplayer?uuid=${uuid}`)).json()
+  
+  let embed = createPlayerEmbed(playerData).setTitle(username).setFooter({ text: `${uuid}\n/v1/hypixel/skyblockplayer` })
+  await interaction.editReply({ content: " ", embeds: [embed]})
+}
+
+async function handleSkyblockItemCommand(client, interaction) {
+  await interaction.reply("Loading...")
+  // Gets the parameters object
+  const parameters = interaction.options
+  const id = parameters.get("id").value
+  await interaction.editReply(`Loading data for ${id}`)
+  
+  const itemData = await getItem(id)
+
+  if (itemData == null) {
+    await interaction.editReply(`No such item found! (${id})`)
+    return
+  }
+  let embed = createItemEmbed(itemData).setTitle(itemData.name).setFooter({ text: `${id}\n/v1/hypixel/skyblockitem` })
+  await interaction.editReply({ content: " ", embeds: [embed]})
+}
+
+function createItemEmbed(itemData) {
+    const embed = new EmbedBuilder()
+    .setColor(config.color)
+
+  embed.addFields(
+    { name: "Item Information:", value: `Name: ${itemData.name}\nRarity: ${itemData.rarity}\nMaterial: ${itemData.material}\nUnstackable: ${itemData.unstackable}\nNPC Sell Price: ${itemData.npcSell} coins` },
+  )
+
+  if (itemData.lowestBin != 0) {
+    embed.addFields({ name: "Auction Information:", value: `Lowest Bin: ${itemData.lowestBin} coins\nAverage Lowest Bin (24 Hours): ${itemData.averageLowestBin} coins` },)
```

Comment:
When using template strings, you can have them in multi lines.
Just replace the \n with an actual linebreak.